packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -99,8 +99,9 @@ class _MyHomePageState extends State<MyHomePage> {
           // arranges them vertically. By default, it sizes itself to fit its
           // children horizontally, and tries to be as tall as its parent.
           //
-          // Invoke "debug painting" (press "p" in the console or choose the
-          // "Toggle Debug Paint" command in Android Studio or Visual Studio Code)
+          // Invoke "debug painting" (press "p" in the console, choose the
+          // "Toggle Debug Paint" action from the Flutter Inspector in Android
+          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
```

Comment:
nit: is it appropriate to call it an action in Visual Studio Code for consistency?

Reply:
I think calling it a command here (in VS Code) makes sense - you get to it by opening the command palette.

Reply:
sgtm